<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReviewWin</class>
 <widget class="QWidget" name="ReviewWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuWin</string>
  </property>
  <widget class="QPushButton" name="pbPageDown">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Page Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSearch">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPageUp">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Page Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPrint">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDelete">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTransmit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Transmit</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
